### DISCUSSION TITLE
Multiple App Identifiers for Android & iOS

### DISCUSSION THREAD

**Discussion by EdwardMcFlurry** [dotnet/maui#16373] [Q&A] [answered]
Hello

I was wondering how to go about setting multiple App Identifiers for both Android and iOS but each platform will need to have App Identifier that is slightly different from the other. Thanks in advance.

**Comment by baaaaif** [comment, marked as answer]
We solved this with multiple PropertyGroups and Conditions in the csproj file.

```
<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)'=='Release_xxx|net7.0-android'">
    <!-- Display name -->
    <ApplicationTitle>xxx</ApplicationTitle>
    <DefineConstants>xxx</DefineConstants>

    <!-- App Identifier -->
    <ApplicationId>de.company.xxx</ApplicationId>
    <ApplicationIdGuid>yyy</ApplicationIdGuid>
  </PropertyGroup>
```

**Comment by EdwardMcFlurry** [comment]
> We solved this with multiple PropertyGroups and Conditions in the csproj file.
> 
> ```
> <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)'=='Release_xxx|net7.0-android'">
>     
>     <ApplicationTitle>xxx</ApplicationTitle>
>     <DefineConstants>xxx</DefineConstants>
> 
>     
>     <ApplicationId>de.company.xxx</ApplicationId>
>     <ApplicationIdGuid>yyy</ApplicationIdGuid>
>   </PropertyGroup>
> ```

@baaaaif thank you for the answer 👍🏻
